ej/models.py

```diff
@@ -8,6 +8,8 @@
 class Problem(db.Entity):
     title = Required(str, unique=True)
     time_limit = Required(int)
+    custom_check = Required(bool)
```

Comment:
As the case where you have a script but don't wanna use can be a very specific application, I will make the modifications and keep it simpler